data/src/main/java/com/microsoft/azure/kusto/data/exceptions/DataClientException.java

```diff
@@ -0,0 +1,20 @@
+package com.microsoft.azure.kusto.data.exceptions;
+
+public class DataClientException extends Exception {
```

Comment:
```suggestion
public class KustoClientException extends Exception {
```

Reply:
We agreed to remove "Kusto" from all class names. It is still in the package names.

Reply:
As above: We agreed to remove "Kusto" from all class names. It is still in the package names.